agent/src/agent/pipeline/config/jython_scripts/events_destination.py

```diff
@@ -17,9 +17,18 @@
 def main():
     for record in sdc.records:
         send_event(record)
+        send_offset(record.value['offset'])
```

Comment:
Just to be sure, if inside `send_event` function `client.send_events` will fail to send with some exception, we still perform `send_offset` after it, right?